README.md

```diff
@@ -1,21 +1,47 @@
+When you try to train a topic model, you’ll quickly realize that you need to make a lot of decisions and that those decisions don’t have a clear “right answer”. But, if you have a way to validate your model, you can explore the range of possible decisions and use this validation step as a way to compare model performance and ultimately help you decide which parameters make sense for you. 
 
-1. Do a better job scraping text articles from the internet. I know that I haven’t figured out all the ways of dealing with punctuation and other formatting issues because some words still get smashed together, when there should be a space between them. I will write some simple tests to check that some of these common “uber words” aren’t created. 
+This pipeline makes it easy to manipulate different parameters (number of topics, text pre-processing steps) and to use an external "test" to pick the best performing model. This "winning" model can then be applied to documents of interest.
 
-2. The purpose of the model validation step is to compare models that vary across a set of parameters. I will create functions that allow the user to set these parameters (number of topics, text preprocessing steps, etc).
+# Steps
 
-3. Currently, I create figures that summarize the results of model comparisons in a separate script. I would like these figures to run and output by default. I think I separated this initially because I was having issues creating figures in the cluster. I’ll add a parameter that will allow the user to turn off this option if they want (e.g., if it avoids issues on a cluster).
+Each step of this process is contained in a separate directory.
 
-4. I want to figure out how to store a model (probably by pickling). Currently I re-train the “winning” model when I apply it to the data that I care about later. I would like to only have to train these models a single time, and store the ones that I want to use later.
+## 1. Scrape Training Data
 
-5. There is currently a LOT of redundancy in these scripts. I will create functions for operations that I currently repeat.
+./scrape_training_data
 
-6. Some of my paths are hard-coded. I will be more generalizable about calling path names.
+Contained in this directory are scripts that find, scrape, and store text from articles in The Dartmouth daily newspaer archives. 
 
-7. I will make these scripts PEP8 compatible.   
\ No newline at end of file
+## 2. Train Models
+
+./train_models
+
+The script 'train_models.py' takes the training data created in Step 1, pre-processes it, and uses it to train a series of topic models that vary across a set of parameters (number of topics, text pre-processing steps). These models are pickled and stored in 'models_pickles'. Each model also has a corresponding text files in 'models_topics' that lists the top words associated with topic. These collections of words allow the user to provide model 'labels' and to quickly assess the interpretability of the topics.
```

Comment:
you might better just call that directory `trained_models` or alike, and just rely how they are stored in the extension of the file, which you already have `.pkl`.  you might later follow scikit-learn's advice and use joblib's functionality to store/load them https://scikit-learn.org/stable/modules/model_persistence.html and store into `.joblib` files.  If you keep working with only one type of the model, creating .json (or some other e.g. .yaml) serialization might help with maintainability, compatibility (pickling could cause troubles across versions of python or even possibly scikit-learn itself) and ability to "see" the model. See e.g. approach taken at https://stackabuse.com/scikit-learn-save-and-restore-models/ which might be a bit more appropriate/lightweight here in comparison to the one you found in https://cmry.github.io/notes/serialize